src/util/CMakeLists.txt

```diff
@@ -1,5 +1,6 @@
 file(GLOB_RECURSE sources "*.cpp" "*.h")
 
+
```

Comment:
🚫 Why is a java-models-library bump in this commit c04d5960b?

Reply:
that's a mistake